test/skimNtuple2016.cpp

```diff
@@ -3753,15 +4130,15 @@ int main (int argc, char** argv)
                                mVisB, pxB, pyB,
                                pxMiss_eledown, pyMiss_eledown,
                                chiA_eledown, chiB_eledown,
-                               desiredPrecisionOnMt2);
+                               desiredPrecisionOnMt2);*/
 
           theSmallTree.m_MT2 = MT2;
-          theSmallTree.m_MT2_tauup = MT2_tauup;
+          /*theSmallTree.m_MT2_tauup = MT2_tauup;
```

Comment:
please do not use `\*...*\` for comments, put `\\` in front of each line, it' easier to read and correct in git.